pkg/skaffold/deploy/helm/helm.go

```diff
@@ -687,12 +692,9 @@ func sortKeys(m map[string]string) []string {
 
 // getArgs calculates the correct arguments to "helm get"
 func getArgs(v semver.Version, releaseName string, namespace string) []string {
-	args := []string{"get"}
-	if v.GTE(helm3Version) {
-		args = append(args, "all")
-		if namespace != "" {
-			args = append(args, "--namespace", namespace)
-		}
+	args := []string{"get", "all"}
+	if namespace != "" {
```

Comment:
ah! looks like we passed the namespace arg only for helm2. :)
Wonder how helm3 users didn't hit this.

Reply:
.... 😲